src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -58,23 +76,36 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
             await this.GeneratePublicNugetSource();
             Directory.CreateDirectory(reportsPath);
 
-            var installSwaggerCliTask = swaggerManager.InstallSwaggerCliAsync(cancellationToken);
-            var installSpectralTask = spectralManager.InstallSpectralAsync(cancellationToken);
-            var installOasdiffTask = oasdiffManager.InstallOasdiffAsync(cancellationToken);
-            
-            await installSwaggerCliTask;
-            await installSpectralTask;
-            await installOasdiffTask;
-
-            var generateOpenApiDocsPath = (await swaggerManager.RunSwaggerAsync(this.OpenApiSwaggerDocumentNames, cancellationToken)).ToList();
-            
-            if (!this.CheckIfBaseSpecExists())
+            switch (this.DevelopmentMode)
             {
-                return false;
+                case CodeFirst:
+                    await codeFirstProcess.Execute(
+                        this.OpenApiSpecificationFiles,
+                        this.OpenApiSwaggerDocumentNames,
+                        this.OpenApiSpectralRulesetUrl,
+                        cancellationToken);
+                    break;
+                
+                case ContractFirst:
+                    var isSuccess = await contractFirstProcess.Execute(
```

Comment:
the bool return value serves no purpose, right?

Reply:
If we return false it set the task as an error

![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/5649ea30-7120-4cc6-a69e-57e9c0566a5f)